chunky/src/java/se/llbit/chunky/renderer/DefaultRenderManager.java

```diff
@@ -246,6 +246,11 @@ public void run() {
         // Wait for a scene state change (e.g. user editing the scene)
         ResetReason reason = sceneProvider.awaitSceneStateChange();
 
+        if (reason == ResetReason.SETTINGS_CHANGED_SOFT) {
+          // Settings sync reset. No need to re-render.
+          continue;
+        }
```

Comment:
How about introducing a new `requiresRerender` to the `ResetReason` and then

```suggestion
        if (!reason.isRerenderRequired()) {
          // Settings sync reset. No need to re-render.
          continue;
        }
```